bindings/matrix-sdk-ffi/src/room.rs

```diff
@@ -810,6 +810,23 @@ impl Room {
 
         Ok(())
     }
+
+    /// Attempt to manually resend messages that failed to send due to issues
+    /// that should now have been fixed. This is useful for example, when
+    /// there's a
```

Comment:
nit: let's add some breathing space:

```suggestion
    /// that should now have been fixed.
    ///
    /// This is useful for example, when there's a
```